README.md

```diff
@@ -5,56 +5,58 @@
 
 
 ## Setup
-Based on python 3.4
 
-Create a virtual env for python3 and make sure that it's activated
+[Install Docker](https://docs.docker.com/engine/installation/)
 
+Install pipenv
+```bash
+pip install pipenv
```

Comment:
I get the following error? 

Installing dependencies from Pipfile.lock (0ce6d1)…
An error occurred while installing psycopg2==2.6.2! Will try again.
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 38/38 — 00:00:03
Installing initially–failed dependencies…
Collecting psycopg2==2.6.2 ▉▉▉▉▉▉▉▉▉▉ 0/1 — 00:00:00
  Using cached psycopg2-2.6.2.tar.gz
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/psycopg2.egg-info
    writing pip-egg-info/psycopg2.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/psycopg2.egg-info/dependency_links.txt
    writing top-level names to pip-egg-info/psycopg2.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/psycopg2.egg-info/SOURCES.txt'
    Error: pg_config executable not found.
    
    Please add the directory containing pg_config to the PATH
    or specify the full executable path with the option:
    
        python setup.py build_ext --pg-config /path/to/pg_config build ...
    
    or with the pg_config option in 'setup.cfg'.
    
    ----------------------------------------

Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/l8/ppxlhm3s58356981xh6xlk880000gp/T/pip-build-ssz2oc7h/psycopg2/

Reply:
This comment should be attached to
 ```bash
pipenv install
```

Reply:
@jwgwalton `brew install postgresql` / `apt-get install libpq-dev`

Reply:
Probably worth updating README with the system dependency.

Reply:
It needs an external dependency, I thought it was all self-contained?

Reply:
http://initd.org/psycopg/docs/install.html#build-prerequisites